graphql-schemas/schema.yml

```diff
@@ -709,6 +709,7 @@
   - { name: resetPasswords, type: AWSAccountResetPassword_v1, isList: true }
   - { name: premiumSupport, type: boolean, isRequired: true }
   - { name: partition, type: string }
+  - { name: terraformState, type: AWSTerraformState_v1, isList: true }
```

Comment:
with everything that is going on in https://github.com/app-sre/qontract-schemas/pull/87 i wonder if we should also support other terraform state backends and reference an interface type instead of a concrete one like [here](https://github.com/app-sre/qontract-schemas/pull/87/files#diff-6bad3cee90319b7494304e3ce4e439992becd03f9d8825eca7f44687f8184ee9R1351) 

cc @maorfr

Reply:
That is a great idea.

Reply:
makes sense to me!